<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmLicense</class>
 <widget class="QMainWindow" name="frmLicense">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>556</width>
    <height>281</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>License Management</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>40</y>
      <width>81</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Host ID</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtPower">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>40</y>
      <width>321</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnRegister">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>210</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Register</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnCancel">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>210</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Cancel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>120</y>
      <width>111</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>License code</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtLicense">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>120</y>
      <width>321</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnBrowser">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>120</y>
      <width>101</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>license...</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
